config.py

```diff
@@ -1,7 +1,13 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+# Deprecate COBOT_ROOT
+BOT_ROOT = os.environ.get('BOT_ROOT', os.getcwd())
+if 'COBOT_ROOT' in os.environ:
+    os.environ.setdefault('BOT_ROOT', 'COBOT_ROOT')
```

Comment:
this will never kick-in, right? `BOT_ROOT` will always have a value, and `os.environ['BOT_ROOT']` is never executed again.

I think also a warning message should be only emitted if `BOT_ROOT` wasn't set, but only `COBOT_ROOT`.

Reply:
The 6th line will kick in if 'COBOT_ROOT' env var is set and will change this env var to 'BOT_ROOT'.
The warning message will be emitted if instance uses 'COBOT_ROOT' instead of 'BOT_ROOT'.

Reply:
Nah there's something wrong. 1st: You are setting the string `'COBOT_ROOT'` itself for the `BOT_ROOT` key inside the environment variables. 2nd: No code will ever call `os.environ['BOT_ROOT']` again, so this is meaningless.

So I played through all cases:
1. Only `BOT_ROOT` was set in env: `BOT_ROOT` var gets set to the env-var value.
2. Only `COBOT_ROOT` was set in env: `BOT_ROOT` is set to `os.getcwd()`, the `BOT_ROOT` key inside env gets the value `'COBOT_ROOT'` (a string) and issues a deprecation warning
3. Both were set in env: `BOT_ROOT` and `COBOT_ROOT`: `BOT_ROOT` is set to env value, the `BOT_ROOT` key is set to `'COBOT_ROOT'` string in `os.environ` and a deprecation warning is issued.

I would only issue a deprecation warning if only `COBOT_ROOT` is present, but not `BOT_ROOT`. Because then the user has missed the env-var change. But if both are present, he might have it there on purpose.